src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -31,6 +31,8 @@
 
 PARAMETER_NUM_UNITS = [" ", "K", "M", "B", "T"]
 UNKNOWN_SIZE = "?"
+NON_LAYER_PARAMS_NAME = "other params"
+NOT_APPLICABLE = "n/a"
```

Comment:
Maybe I should just use unknown size for in size and out size?

Reply:
Need advice on naming conventions

Reply:
Only reason I did not use UNKNOWN_SIZE is cause the left params are not a layer so they would not have any in size or out size concept

Reply:
I think both `N/A` or `-` is fine in this case :)